agent/consul/acl.go

```diff
@@ -211,6 +212,40 @@ type ACLResolverConfig struct {
 	Tokens *token.Store
 }
 
+// TODO: rename the fields to remove the ACL prefix
+type ACLResolverSettings struct {
+	ACLsEnabled  bool
+	Datacenter   string
+	NodeName     string
+	ACLPolicyTTL time.Duration
+	ACLTokenTTL  time.Duration
+	ACLRoleTTL   time.Duration
```

Comment:
Why not carry the comments over from their prior home?

Reply:
Oh ya, I should do that. I missed it for the first few fields I did. I'll do that now